client/src/app/shared/components/sorting-list/sorting-list.component.scss

```diff
@@ -1,6 +1,5 @@
 .list {
-    width: 75%;
-    max-width: 100%;
+    width: 100%;
```

Comment:
might be dangerous, keep in mind that touch-scrolling does not work and dragable lists lists.